synapse/cortex.py

```diff
@@ -5509,8 +5606,15 @@ async def feedFromAxon(self, sha256, opts=None):
             async def fill():
                 nonlocal feedexc
                 try:
-
+                    first = True
                     async for item in self.axon.iterMpkFile(sha256):
+                        if first:
+                            try:
+                                s_schemas.reqValidExportStormMeta(item)
```

Comment:
We should probably do a meta version check here? Maybe a synapse version check too?